editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -1396,8 +1396,7 @@ void TileMapEditor::forward_canvas_draw_over_viewport(Control *p_overlay) {
 		return;
 
 	Transform2D cell_xf = node->get_cell_transform();
-
-	Transform2D xform = p_overlay->get_canvas_transform() * node->get_global_transform();
+	Transform2D xform = CanvasItemEditor::get_singleton()->get_canvas_transform() * node->get_global_transform();
```

Comment:
You should be able to use `canvas_item_editor->get_canvas_transform()` directly here.

Reply:
Indeed, that should be more consistent, I agree.

Reply:
Ping.